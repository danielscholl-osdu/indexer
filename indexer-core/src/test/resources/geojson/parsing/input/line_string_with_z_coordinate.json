{
    "type": "FeatureCollection",
    "bbox": null,
    "features": [
        {
            "type": "Feature",
            "bbox": null,
            "geometry": {
                "type": "LineString",
                "bbox": null,
                "coordinates": [
                    [
                        34.51305348816962,
                        0.035298719063461145,
                        7.0
                    ],
                    [
                        34.513062650895186,
                        0.030790856372375844,
                        7.0
                    ],
                    [
                        34.513071785973196,
                        0.0262829935696415,
                        7.0
                    ]
                ]
            },
            "properties": {}
        }
    ],
    "properties": null,
    "persistableReferenceCrs": null,
    "persistableReferenceUnitZ": {
        "scaleOffset": {
            "scale": 1.0,
            "offset": 0.0
        },
        "symbol": "m",
        "baseMeasurement": {
            "ancestry": "Length",
            "type": "UM"
        },
        "type": "USO"
    }
}