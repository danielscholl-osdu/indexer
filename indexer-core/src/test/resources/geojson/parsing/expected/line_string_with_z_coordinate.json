{
    "type": "geometrycollection",
    "geometries": [
        {
            "type": "linestring",
            "coordinates": [
                [
                    34.51305348816962,
                    0.035298719063461145,
                    7.0
                ],
                [
                    34.513062650895186,
                    0.030790856372375844,
                    7.0
                ],
                [
                    34.513071785973196,
                    0.0262829935696415,
                    7.0
                ]
            ]
        }
    ]
}